lib/npg_qc/autoqc/checks/rna_seqc.pm

```diff
@@ -0,0 +1,328 @@
+package npg_qc::autoqc::checks::rna_seqc;
+
+use Moose;
+use namespace::autoclean;
+use English qw( -no_match_vars );
+use Carp;
+use File::Spec::Functions qw( catdir );
+use File::Basename;
+use File::Path qw(make_path);
+use npg_qc::autoqc::types;
+use npg::api::run;
+use Readonly;
+use Try::Tiny;
+use DateTime;
+use npg_tracking::util::types;
+
+extends qw(npg_qc::autoqc::checks::check);
+
+with qw(npg_tracking::data::reference::find 
+        npg_common::roles::software_location
+        npg_tracking::data::transcriptome::find
+       );
+
+our $VERSION = '0';
+
+Readonly::Scalar our $EXT => q[bam];
+Readonly::Scalar my $RNASEQC_JAR_NAME       => q[RNA-SeQC_v1.1.8.jar];
```

Comment:
Can we avoid having version in the jar name? As it is, impossible to update the jar without changing the source code.